test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeUnsafeTests.java

```diff
@@ -284,7 +278,6 @@ static public void testUninitializedDefaultValueOfArray() throws Throwable {
 
 	@Test
 	static public void testuninitializedVTClassHasNullDefaultValue() {
-		@ImplicitlyConstructible
```

Comment:
It looks like Unsafe.uninitializedDefaultValue has not existed upstream for some time. I think this entire test and Unsafe.uninitializedDefaultValue can be removed.